crates/ruma-api-macros/src/api/metadata.rs

```diff
@@ -125,32 +166,43 @@ enum Field {
     Path,
     RateLimited,
     Authentication,
+    Added,
+    Deprecated,
+    Removed,
+}
+
+macro_rules! map_kw {
```

Comment:
Found an opportunity to make this less cumbersome.

Or, alternatively;

Eyo, i heard you like macros, so put a macro that works with macros in your macro.